docs/docs/prompts/index.md

```diff
@@ -0,0 +1,145 @@
+# Prompts
+
+Prompts are instructions for Large Language Models (LLMs) that guide them in generating responses.
+They define the content and structure of your interactions with LLMs.
+This section describes how to create and run prompts with Koog.
+
+## Creating prompts
+
+In Koog, all prompts are represented as [**Prompt**](https://api.koog.ai/prompt/prompt-model/ai.koog.prompt.dsl/-prompt/index.html)
+objects. A Prompt object contains:
+
+- **ID**: A unique identifier for the prompt.
+- **Messages**: A list of messages that represent the conversation with the LLM.
+- **Parameters**: Optional [LLM configuration parameters](https://api.koog.ai/prompt/prompt-model/ai.koog.prompt.params/-l-l-m-params/index.html)
+  (such as temperature, tool choice, and others).
+
+All Prompt objects are structured prompts defined using the Kotlin DSL, which lets you specify the structure of the conversation.
+
+!!! note
+    AI agents let you provide a simple text prompt instead of creating a Prompt object.
+    They automatically convert the text prompt to the Prompt object and send it to the LLM for execution.
+    This is useful for a [basic agent](basic-agents.md) that only needs to run a single request.
+
+
+<div class="grid cards" markdown>
```

Comment:
These cards make it look like there are two types of prompts: structured and multimodal. But if I am not mistaken, the latter is actually just an extension of user messages to attach images, audio, video, or arbitrary files, right? Why is it called multimodal?
Suggestions:
- Name it something else (unless I am missing why "multimodal input" is correct and informative)
- Put it under `Structured prompts` (since it is an extension of the prompt DSL described there) or keep it on the same level if you don't feel like increasing the nesting for just one page
- Remove the cards